star-ui/src/App.tsx

```
@@ -7,44 +9,195 @@ import { OdometryPanel } from './components/OdometryPanel';
+                className="glass-panel panel-hover-container stagger-fade-in"
+                style={{
+                  animationDelay: `${0.05 + i * 0.05}s`,
+                }}
+              >
+                {/* Maximize button in header */}
+                <button
+                  className="panel-maximize-btn"
+                  onClick={() => toggleMaximize(key)}
+                  title="Maximize panel"
+                  aria-label={`Maximize ${key} panel`}
+                >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)